pan-domain-auth-verification/src/main/scala/com/gu/pandomainauth/Settings.scala

```diff
@@ -32,7 +32,7 @@ trait FailureWithCause extends SettingsFailure {
 }
 
 case class SettingsDownloadFailure(cause: Throwable) extends FailureWithCause {
-  override val description: String = "Unable to download public key"
+  override val description: String = "Unable to access settings file"
```

Comment:
This change gives a less misleading error message when the user runs the script supplying a path that doesn't point to a file:

```
sbt key-rotation/run /tmp/file-does-not-exist.settings
Loading current conf from /tmp/file-does-not-exist.settings
Unable to access settings file
```